CMakeLists.txt

```diff
@@ -63,6 +64,23 @@ include(ExternalProject)
 include(CheckCXXCompilerFlag)
 include(CheckSymbolExists)
 
+if(EMSCRIPTEN)
+  # FIXME: As of writing, Emscripten fails with "wasm-ld: error: /path/to/emsdk/upstream/emscripten/cache/sysroot/lib/wasm32-emscripten/thinlto/libc.a(fileno.o): attempt to add bitcode file after LTO (fileno)"
+  # Seems to be a known issue: https://github.com/emscripten-core/emscripten/issues/20275
+  option(SUPERTUX_IPO "Use interprocedural optimisation (Takes more RAM at compilation, gives smaller and faster builds)" OFF)
+else()
+  option(SUPERTUX_IPO "Use interprocedural optimisation (Takes more RAM at compilation, gives smaller and faster builds)" ON)
```

Comment:
Does "faster builds" mean the compiled program is faster or the compilation is faster?

Reply:
Compilation is faster, 50-66% less time on my system. I've hesitated to put an explanation at all, but I thought it might help less familiar people understand the reasons. I'm open to changing the wording if necessary.

Reply:
"gives smaller and faster builds" sounds ambiguous to me because "builds" can refer to the output of the build process or the build process itself, as far as I know.
Maybe "reduces program size and compilation time" can be used instead if the statement is true.

Reply:
My apologies, I confused PCH and IPO. PCH is what makes compilation much faster, LTO takes more time to compile and gives better binaries, both in terms of runtime speed and executable size.

Reply:
I've clarified the wording with the latest commit.